Cargo.toml

```diff
@@ -39,5 +40,6 @@ version = "0.3.0-alpha"
 features = ["compat"]
 
 [dependencies.screeps-api]
-version = "0.5"
+version = "0.6"
 git = "https://github.com/daboross/rust-screeps-api.git"
+tag = "screeps-api-0.6.0"
```

Comment:
If there are things in `screeps-api` which currently don't build, I think we should fix them there rather than avoiding the update here.

In particular, we're going to need an updated screeps-api to fix #1.

Reply:
Oh, just built locally and I see why you did this. I... don't entirely understand how the `screeps-api-0.6.0` tag would work, though, when we're still listing the version dependency as `0.5`?

I think long-term this tag should be removed, but if for now it fixes the build I'm OK with merging it until we update to the later screeps-api version :+1:

Reply:
It actually doesn't build with tag 0.5 either! What I suspect happened was that since these were developed concurrently, the git HEAD that was supposedly version 0.5 was actually a dev version of 0.6. Which is why I bumped it, because it builds if I do that. The version given to cargo can be increased though.

Reply:
Ah, that makes sense! I somehow forgot you'd also bumped the version when I was trying this out locally and the writing the second comment.